src/apps/search-result/facet-browser.test.ts

```diff
@@ -0,0 +1,247 @@
+const coverUrlPattern = /^https:\/\/res\.cloudinary\.com\/.*\.(jpg|jpeg|png)$/;
+
+describe("Facet Browser", () => {
+  // I have used it.only because it is best practice to have self contained tests
+  // https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests
+  it.only("Open facet browser and check if all the facets are there", () => {
+    cy.contains(".disclosure__text", "Main languages");
+    cy.contains(".disclosure__text", "Access types");
+    cy.contains(".disclosure__text", "Children or adults");
+    cy.contains(".disclosure__text", "Creators");
+    cy.contains(".disclosure__text", "Fiction or nonfiction");
+    cy.contains(".disclosure__text", "Genre and form");
+    cy.contains(".disclosure__text", "Material types");
+    cy.contains(".disclosure__text", "Subjects");
+    cy.contains(".disclosure__text", "Work types");
```

Comment:
Please [do not select against CSS classes as they are likely to change](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements).

I think it would be OK to select based on the text content alone. If we want to be more specific then [selecting based on accessibility roles is also a good idea](https://testing-library.com/docs/queries/about/#priority).

This goes for the entire test.